packages/text-editor/src/components/ReferenceInput.svelte

```diff
@@ -23,15 +23,18 @@
   import TextEditor from './TextEditor.svelte'
 
   const dispatch = createEventDispatcher()
+  let content: string
```

Comment:
I need this one to be `export let content: string` to be able to use ReferenceInput to edit comments.